### PR TITLE
feat(connections): add icon name on GET return results

### PR DESCRIPTION
**Pull request by rogefm**
Add Icon_name on GET return results

### CODE REVIEW

File: gateway/connection/service.go [hunk 1]
```diff
@@ -30,12 +30,13 @@ type (
...
 	BaseConnection struct {
-		Id             string         `json:"id"       edn:"xt/id"`
-		Name           string         `json:"name"     edn:"connection/name"    binding:"required"`
-		Command        []string       `json:"command"  edn:"connection/command" binding:"required"`
-		Type           Type           `json:"type"     edn:"connection/type"    binding:"required"`
-		AgentId        string         `json:"agent_id" edn:"connection/agent"   binding:"required"`
-		SecretProvider SecretProvider `json:"-"        edn:"connection/secret-provider"`
+		Id             string         `json:"id"       	edn:"xt/id"`
+		Name           string         `json:"name"     	edn:"connection/name"      	binding:"required"`
+		IconName       string         `json:"icon_name"	edn:"connection/icon_name"`
```
Comment (danielbonilha) [inline gateway/connection/service.go]: for the edn part of the tags, we are using kebab-case... this means that the edn should be "connection/icon-name"

File: gateway/connection/storage.go [hunk 2]
```diff
@@ -19,7 +19,8 @@ type (
 	xtdb struct {
 		Id             string         `edn:"xt/id"`
 		OrgId          string         `edn:"connection/org"`
-		Name           string         `edn:"connection/name" `
+		Name           string         `edn:"connection/name"`
+		IconName       string         `edn:"connection/icon_name"`
```
Comment (danielbonilha) [inline gateway/connection/storage.go]: for the edn part of the tags, we are using kebab-case... this means that the edn should be "connection/icon-name"

File: gateway/connection/service.go (hunk 1 above)
Comment (rogefm) [reply gateway/connection/service.go]: Done!

File: gateway/connection/storage.go (hunk 2 above)
Comment (rogefm) [reply gateway/connection/storage.go]: Done!

**Pull request merged**